arbnode/transaction_streamer.go

```diff
@@ -1715,33 +1701,11 @@ func (s *TransactionStreamer) checkEspressoLiveness() error {
 	log.Warn("enabling the escape hatch, hotshot is down")
 	s.EscapeHatchEnabled = true
 
-	// Skip the espresso verification for the submitted messages
-	submitted, err := s.getEspressoSubmittedPos()
-	if err != nil {
-		return err
-	}
-	if len(submitted) == 0 {
-		return nil
-	}
-
-	s.espressoTxnsStateInsertionMutex.Lock()
-	defer s.espressoTxnsStateInsertionMutex.Unlock()
-
-	batch := s.db.NewBatch()
-	err = s.cleanEspressoSubmittedData(batch)
-	if err != nil {
-		return err
-	}
-	err = batch.Write()
-	if err != nil {
-		return err
-	}
-
```

Comment:
Previously I thought we needed to clean the submitted data after detecting hotshot failure was because it would cause a dead loop if hotshot has lost some transactions. With the resubmission implementation, this wasn't a case.

Now if the escape hatch is on and later off, the batch poster will veify the submitted messages from an early position even though some of the messages were already added to a batch. Still thinking how to test this scenario thoroughly.

Reply:
Tbh I'm not even thinking about the escape hatch since Rari isn't using it. I don't think we should let it block this PR and the release timeline. But keep thinking about how to test it since we might reenable it for other integrations.

Reply:
yea even I think it should not be a concern right now!